tests/e2e/tests/critical-path/browser/keylist-actions.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { acceptLicenseTermsAndAddDatabaseApi } from '../../../helpers/database';
+import { BrowserPage } from '../../../pageObjects';
+import {
+    commonUrl,
+    ossStandaloneConfig
+} from '../../../helpers/conf';
+import { rte } from '../../../helpers/constants';
+import { deleteStandaloneDatabaseApi } from '../../../helpers/api/api-database';
+import { Common } from '../../../helpers/common';
+
+const browserPage = new BrowserPage();
+const common = new Common();
+
+const keyName = common.generateWord(10);
+fixture `Actions with Key List on Browser page`
+    .meta({type: 'critical_path', rte: rte.standalone})
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .afterEach(async() => {
+        // Delete database
+        await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    });
+test('Verify that user can delete key in List mode', async t => {
+    // Add new key
```

Comment:
its better to set keyName for each test differently
keyName = common.generateWord(10);

in case if one test will fail on deletion - the second will fail too, because it won't be able to create a key with the same name